lms/static/scripts/frontend_apps/errors.ts

```diff
@@ -150,6 +151,7 @@ export function isLTILaunchServerError(error: ErrorLike): error is APIError {
       'canvas_student_not_in_group',
       'vitalsource_user_not_found',
       'vitalsource_no_book_license',
+      'student_not_in_course',
```

Comment:
Is it possible to check if the string is part of the union type? 

A light search suggests it's not trivial but I'm not even sure what I'm looking for.